src/plugins/lua/weechat-lua.c

```diff
@@ -560,32 +558,45 @@ weechat_lua_load (const char *filename, const char *code)
         return NULL;
     }
 
-#ifdef LUA_VERSION_NUM /* LUA_VERSION_NUM is defined only in lua >= 5.1.0 */
-    luaL_openlibs (lua_current_interpreter);
-#else
-    luaopen_base (lua_current_interpreter);
-    luaopen_string (lua_current_interpreter);
+#ifndef LUA_VERSION_NUM
+    luaopen_base  (lua_current_interpreter);
+    luaopen_string(lua_current_interpreter);
     luaopen_table (lua_current_interpreter);
-    luaopen_math (lua_current_interpreter);
-    luaopen_io (lua_current_interpreter);
+    luaopen_math  (lua_current_interpreter);
+    luaopen_io    (lua_current_interpreter);
     luaopen_debug (lua_current_interpreter);
-#endif /* LUA_VERSION_NUM */
+#else
+    luaL_openlibs (lua_current_interpreter);
+#endif /* LUA_VERSION_NUM */  
 
     weechat_lua_register_lib (lua_current_interpreter, "weechat",
                               weechat_lua_api_funcs,
                               weechat_lua_api_consts);
 
-#ifdef LUA_VERSION_NUM
-    if (luaL_dostring (lua_current_interpreter, lua_redirect_output) != 0)
-#else
-    if (lua_dostring (lua_current_interpreter, lua_redirect_output) != 0)
-#endif /* LUA_VERSION_NUM */
-    {
-        weechat_printf (NULL,
-                        weechat_gettext ("%s%s: unable to redirect stdout "
-                                         "and stderr"),
-                        weechat_prefix ("error"), LUA_PLUGIN_NAME);
+    // Remove references to stdout and stderr
+    
+    lua_getglobal(lua_current_interpreter, "io");
+    if (lua_istable(lua_current_interpreter, -1)) {
+        // io.{stdout,stderr} = nil
```

Comment:
Please remove any commented code.

Reply:
It is not commented code, it is the Lua code "equivalent" of the following API calls, it is considered standard practice when performing API calls.